docker-compose/docker-compose-cluster.yml

```diff
@@ -0,0 +1,96 @@
+version: '3.4'
```

Comment:
I don't get the point of this file, you could achieve the same enabling cluster plugin and using docker-compose `scale` capacity (https://github.com/kuzzleio/kuzzle-plugin-prometheus/blob/master/demo/docker-compose.yml). To make the Kuzzle service scale to 3 nodes for example use this:
```
docker-compose up --scale kuzzle=3
```

Reply:
Docker Compose services provides native round robin between service container(s)

Reply:
Oh I didn't know that in fact I thought that cluster port will conflict if you scale the docker compose.
But the purpose of this file is to be able to target each node independently for testing purpose, which you can easily achieve by changing the destination port when you use Kourou for example

Reply:
The best thing would be to have nginx doing the round Robin on each node and still exposing each node port.
in this way we can have a real cluster endpoint and also the possibility to target each node independently

Reply:
With `scale`, you have capabilities to request each node with its docker local IP

Reply:
IMO, the project root Docker Compose file is sufficient to tests all the possible scenario. Maybe the only thing to add is a complete documentation in README or CONTRIBUTING to know how to start Kuzzle cluster or Kuzzle with HTTPS for example

Reply:
Yes but then I have to loop into the IP of each Kuzzle node and then type the 12 numbers of the IP to target a node which is not very convenient compare to only change the port which is fixed.
The root docker compose it's sufficient but not convenient IMHO

Reply:
About the cluster explanation I prefer to wait the cluster integration into the core to update the CONTRIBUTING and README